File: SofiForce.Models/Models/EntityModels/PromtionCriteriaSalesManModel.cs
using System;

namespace Models
{
    public class PromtionCriteriaSalesManModel
    {
        public int SalesManCriteriaId { get; set; }
        public int? PromotionId { get; set; }
        public int? SalesManAttributeId { get; set; }
        public string SalesManAttributeCode { get; set; }

        
        public string ValueFrom { get; set; }

        public bool? Excluded { get; set; }
        public bool? IsActive { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/PromtionCriteriaSalesManModel.ts
export interface PromtionCriteriaSalesManModel {
    salesManCriteriaId: number;
    promotionId: number | null;
    salesManAttributeId: number | null;
    salesManAttributeCode: string | null;

    valueFrom: string;

    excluded: boolean | null;
    isActive: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}